src/components/Item/Item.module.css

```diff
@@ -0,0 +1,74 @@
+
+.itemWrap {
+    width: 90%;
+    min-height: 30px;
+    margin-bottom: 10px;
+    padding: 5px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid grey;
+}
+
+.label {
+    flex: 1;
+    font-size: 14px;
+    margin-bottom: 0;
+    font-weight: normal;
+}
+
+.checkbox {
+    position: absolute;
+    display: none;
+}
+.label {
+    position: relative;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+}
+.label:before {
+    content: '';
+    position: absolute;
+    left: 0;
+    width: 22px;
+    height: 22px;
+    background: lightgrey;
+    border-radius: 11px;
+}
+
+.label__comp:before {
+    background: mediumseagreen;
+}
+
+.itemText {
+    margin: 0;
+}
+
+.task {
+    flex: 10;
+    display: flex;
+    font-size: 18px;
+
+}
+
+.done {
+    font-style: italic;
+    text-decoration: line-through;
+    color: grey;
+    opacity: .5;
+}
+
+.btn {
+    width: 30px;
+    height: 30px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/alde_react/pullRequest?prid=5576104)